### ISSUE TITLE
Pass additional flags to k8s components

### ISSUE THREAD

**Issue by Noksa** [rancher-sandbox/rancher-desktop#1519]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://github.com/rancher-sandbox/rancher-desktop/issues) for a feature request that matches the one I want to file, without success.

### Problem Description

Currently I don’t see a native way to specify additional flags to different k8s components.

In my case I want to use all ports for NodePort and it is done via flag --service-node-port-range.

Another useful case is enable feature gates https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/#overview

### Proposed Solution

A way to pass additional flags to k8s components via GUI

### Additional Information

Rancher Desktop 1.0.1

**Comment by ericpromislow** [comment]
I don't think this applies. Rancher Desktop uses [`k3s`](https://k3s.io/) to manage its local kubernetes cluster, and `-service-node-port-range` is a `kubeadm` option (as far as I know).

See #1417 for more information on how to pass arguments to k3s.

Leaving this issue open for now in case there's more information.

**Comment by Noksa** [comment]
Yeah, but in the UI we have section “Kubernetes Settings” and we can’t change its settings from there?

Looks strange.

**Comment by micahmo** [comment]
+1 to this issue. It absolutely applies to K3s, which can pass along api server arguments like `--service-node-port-range` using the `--kube-apiserver-arg` argument. (See [here](https://github.com/k3s-io/k3s/issues/444#issuecomment-751653917).)

For Rancher Desktop on WSL, this can be configured in `/etc/rc.conf` by editing the `K3S_EXEC` entry to look like this:

`K3S_EXEC="--kube-apiserver-arg service-node-port-range=1-65535"`

The `K3S_EXEC` variable was helpefully added in #926 for things just like this, but it would be even better if we could configure it via the UI, such as suggested in #578.
